cve_bin_tool/helper_script.py

```diff
@@ -315,6 +315,76 @@ def output(self):
 
         self.CONSOLE.rule()
 
+    @staticmethod
+    def output_common(common_strings, product_name):
```

Comment:
Can we get a test for `output_common` and `scan_files` please?  And since this is new code: it would be great if we had type hints for these new functions too.

Reply:
@terriko On a second note, output_common and scan_files do not return anything, and just output to the console, how do I approach tests for them?

Reply:
Take a look at how we handle the cli tests -- pytest has a way to capture output and compare it.  I'll warn you that the output capture doesn't work in all areas where we run tests (which is why we prefer to check returns), but I think helper script should be straightforward enough to use the same method.

Reply:
Will look into this, thanks.